samples/tab-ui-templates/ts/src/index.tsx

```diff
@@ -6,14 +6,13 @@ import reportWebVitals from "./report-web-vitals";
 import { HashRouter } from "react-router-dom";
 
 // Initialize the Microsoft Teams SDK
-microsoftTeams.app.initialize().then(() => {
```

Comment:
We followed the all sample, including microsoftTeams.app.initialize().then(() => {. Could you confirm why it’s not working?

Reply:
Yes because, Microsoft Teams SDK requires the app to run inside the **Teams environment.**  

Below method of **initialization** is working fine in **Teams**
**microsoftTeams.app.initialize().then(() => {.})**

Whereas M365, is an outside of Teams environment so it is blocking SDK Execution and giving error like **app initialization failed**, 
And working fine after initializing using below method.
**microsoftTeams.app.initialize()**